setup.py

```diff
@@ -56,6 +56,10 @@ def project_path(*names):
         "console_scripts": [
             "sbomnix  = sbomnix.main:main",
             "nixgraph = nixgraph.main:main",
+            "nix_outdated = scripts.nixupdate.nix_outdated:main",
+            "vulnxscan = scripts.vulnxscan.vulnxscan:main",
+            "repology_cli = scripts.repology.repology_cli:main",
+            "repology_cve = scripts.repology.repology_cve:main"
```

Comment:
Suggest: add a comma at the end of line.

`make black` as well as `make pre-push` (which also runs black) applies this reformatting.